apis/phabricator.py

```diff
@@ -14,4 +14,16 @@ def __init__(self, config):
 
     @logEntryExit
     def submit_patch(self):
-        self.run(["arc", "diff", "--verbatim"])
+        ret = self.run(["arc", "diff", "--verbatim"])
```

Comment:
I suppose when we have the option not to use arc, we will cut this (and other usages of arc in-tree, like https://searchfox.org/mozilla-central/source/taskcluster/docker/python-dependency-update/scripts/update_pipfiles.sh#62 ) over to a non-arc based workflow within the transition window.  We'll probably support using arc to talk to phabricator indefinitely because... that what this proposed server-side thing will use.